### DISCUSSION TITLE
Can the BLE audio examples run on an ESP32-C3 or be built for native_posix?

### DISCUSSION THREAD

**Discussion by twaclaw** [zephyrproject-rtos/zephyr#53804] [Q&A]
Hi there,
I am trying to build and run some BLE audio examples, for instance, `west build -b esp32c3_devkitm samples/bluetooth/unicast_audio_server/`; however, I get some warnings and Zephyr fails to boot after flashing.

`zephyrproject/modules/hal/espressif/components/esp_phy/src/phy_init.c:678:5: warning: implicit declaration of function 'phy_bbpll_en_usb' [-Wimplicit-function-declaration]
  678 |     phy_bbpll_en_usb(true);
      |     ^~~~~~~~~~~~~~~~
`

Are there any limitations on this board to running these examples?

 I also tried to build for `native_posix` and pass a device to the executable (in Linux) but I keep getting errors.

`sudo systemctl stop bluetooth
 sudo build/zephyr/zephyr.exe --bt-dev=hci0`

`
*** Booting Zephyr OS build zephyr-v3.2.0-3518-g7b5780e00764 ***
[00:00:00.400,000] <inf> bt_hci_core: bt_dev_show_info: Identity: 04:EA:56:3C:6F:67 (public)
[00:00:00.400,000] <inf> bt_hci_core: bt_dev_show_info: HCI: version 5.1 (0x0a) revision 0x0100, manufacturer 0x0002
[00:00:00.400,000] <inf> bt_hci_core: bt_dev_show_info: LMP: version 5.1 (0x0a) subver 0x0100
Bluetooth initialized
Failed to set periodic advertising parameters (err -95)
`

Any guidance is much appreciated. 

Thanks in advance.

**Comment by icatorze** [comment]
Hi @twaclaw, I faced same warning here ( warning: implicit declaration of function 'phy_bbpll_en_usb') but apparently this is not a real problem. I wrote a line: extern void phy_bbpll_en_usb(bool en); on the top of phy_init.c file and warning was gone.

For me, no one of BLE neither WIFI examples run on esp32c3. Application hangs on line: register_chipv7_phy(init_data, cal_data, PHY_RF_CAL_FULL); of this phy_init.c file.

When you did run audio example on native_posix, this "Fail on advertising" is not relate to problem that happen to esp32c3.

**Comment by twaclaw** [comment]
Thanks a lot for the answer @icatorze.
That's unfortunate that the examples don't work on the esp32c3.
The native_posix messages are unrelated. 
Best,
